src/status_im/new_group/screen_private.cljs

```diff
@@ -96,4 +96,7 @@
        [sticky-button (label :t/save)
         (if (= group-type :contact-group)
           #(dispatch [:create-new-group group-name])
-          #(dispatch [:create-new-group-chat group-name]))])]))
+          #(dispatch [:create-new-group-chat group-name]))
+        ;; once? set to true, so once component is mounted, on-press handler
```

Comment:
do we really need these comments? i think we can understand this from `sticky-button` function